lib/PullRequestAction.ts

```diff
@@ -0,0 +1,28 @@
+import { OctokitAction } from './OctokitAction';
+
+export abstract class PullRequestAction extends OctokitAction {
+  protected abstract processJiraIssue(issueId: string): Promise<void>;
+
+  protected async execute(): Promise<void> {
+    const issueIds = await this.fixedJiraIssues();
+
+    if (issueIds.length === 0) {
+      console.warn('No Jira issue found in the PR title.');
+    } else {
+      for (const issueId of issueIds) {
+        await this.processJiraIssue(issueId);
+      }
+    }
+  }
+
+  private async fixedJiraIssues(): Promise<string[]> {
+    const pr = await this.getPullRequest(this.payload.pull_request.number);
+
+    if (pr == null) {
```

Comment:
After the other changes, this could be reduced to return with ternary

Reply:
... actually reverting bach to the original `return ... ? []`. That one is better than ternary